[2024-05-31T10:54:58.690+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-31T10:54:30.383878+00:00 [queued]>
[2024-05-31T10:54:58.699+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-31T10:54:30.383878+00:00 [queued]>
[2024-05-31T10:54:58.699+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-05-31T10:54:58.699+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-05-31T10:54:58.699+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-05-31T10:54:58.711+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): set_up_database> on 2024-05-31 10:54:30.383878+00:00
[2024-05-31T10:54:58.715+0000] {standard_task_runner.py:55} INFO - Started process 887 to run task
[2024-05-31T10:54:58.717+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'conference_dag', 'set_up_database', 'manual__2024-05-31T10:54:30.383878+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/conference_dag.py', '--cfg-path', '/tmp/tmpebc_z2x_']
[2024-05-31T10:54:58.717+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask set_up_database
[2024-05-31T10:54:58.765+0000] {task_command.py:388} INFO - Running <TaskInstance: conference_dag.set_up_database manual__2024-05-31T10:54:30.383878+00:00 [running]> on host 312e62f5d623
[2024-05-31T10:54:58.822+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=conference_dag
AIRFLOW_CTX_TASK_ID=set_up_database
AIRFLOW_CTX_EXECUTION_DATE=2024-05-31T10:54:30.383878+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-31T10:54:30.383878+00:00
[2024-05-31T10:54:58.823+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-05-31T10:54:58.824+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'set -e; python /opt/***/mysql_utils/set_up_database.py']
[2024-05-31T10:54:58.832+0000] {subprocess.py:86} INFO - Output:
[2024-05-31T10:54:59.206+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-31T10:54:59.225+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=conference_dag, task_id=set_up_database, execution_date=20240531T105430, start_date=20240531T105458, end_date=20240531T105459
[2024-05-31T10:54:59.250+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-05-31T10:54:59.273+0000] {taskinstance.py:2578} INFO - 3 downstream tasks scheduled from follow-on schedule check
